Unix/installbuilder/datafiles/AIX.data

```diff
@@ -74,3 +87,10 @@ ${{SERVICE_SCRIPT}} start
 #include OmiService_funcs
 StopOmiService
 RemoveOmiService
+
+%Postuninstall_100
+egrep "^omi:" /etc/group 2>&1 >/dev/null
```

Comment:
On Linux, this wouldn't work, as %Postuninstall of the OLD kit may run, damaging the new installation. I think that @johnkord should review this was well, to try and avoid nasty installer bugs.

I'm not sure how postrm/postuninstall works on the UNIX systems, but we should look at that (or try this code) to make sure we don't damage a new installation here.

In particular, look [here](https://github.com/Microsoft/omi/blob/master/Unix/installbuilder/datafiles/Linux_DPKG.data#L2) for an example of what we need to do on Linux platforms.